contrib/build-ceph-base.sh

```diff
@@ -75,7 +75,7 @@ for flavor in $flavors_to_build; do
     make_cmd="make --directory="${SCRIPT_DIR}/.." \
         FLAVORS="${ceph_codename}-${version_without_build},${ceph_container_distro_dir},${distro_release}" \
         IMAGES_TO_BUILD=daemon-base \
-        TAG_REGISTRY="${PUSH_LIBRARY}" \
```

Comment:
I don't think this should be removed unless it is actively causing problems. If the PUSH_LIBRARY is changed in the settings, then this script will no longer work as expected.